.github/workflows/release.yml

```diff
@@ -113,6 +113,8 @@ jobs:
       - name: Upload Release Assets
         run: |
           gh release upload ${{ github.ref_name }} dist/*
+    env:
+      GH_TOKEN: ${{ github.token }}
```

Comment:
If this still bombs, it _might_ be because this needs to be defined at the step level. Could try as-is or move this inside the upload step preemptively :shrug: 

I'm basing this on the note from [here](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context):

> Note: This context property is set by the Actions runner, and is only available within the execution steps of a job. Otherwise, the value of this property will be null.

And the way the [docs](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows) suggest calling gh cli commands:

> For each step that uses GitHub CLI, you must set an environment variable called GH_TOKEN to a token with the required scopes.

Reply:
`env` inherits downwards, I'm validating locally with actionlint incidentally re valid key.

Reply:
Sounds good, I wasn't sure 👍

Reply:
also fwiw, I've been using click's yaml as a reference for this. https://github.com/pallets/click/blob/main/.github/workflows/publish.yaml

Reply:
I went ahead and pushed it down to the step fwiw